python/ray/train/v2/tests/test_collectives.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+import ray
+import ray.train.collectives
+from ray.train.v2.api.data_parallel_trainer import DataParallelTrainer
+
+
+def test_barrier():
+    @ray.remote
```

Comment:
need to add some `ray_start_4_cpus` fixtures to these

Reply:
Done - out of curiosity what is the default behavior when I omit this? I noticed that the test still works with actors on different processes.